lib/openai/http.rb

```diff
@@ -78,6 +78,7 @@ def conn(multipart: false)
         f.use MiddlewareErrors if @log_errors
         f.response :raise_error
         f.response :json
+        f.adapter :async_http if @async
```

Comment:
This will require users to include `async-http-faraday` in their app themselves.